snips/run.sh

```diff
@@ -36,24 +53,30 @@ if [ "$MQTT_BRIDGE" == "true" ]; then
           echo "password $PASSWORD"
       } >> /etc/mosquitto.conf
     fi
-    
+
     {
-        echo "topic # OUT"
-        echo "topic # IN hermes/ hermes/"
+        echo "topic hermes/intent/# out"
+        echo "topic hermes/hotword/toggleOn out"
+        echo "topic hermes/hotword/toggleOff out"
+        echo "topic hermes/asr/stopListening out"
+        echo "topic hermes/asr/startListening out"
+        echo "topic hermes/nlu/intentNotParsed out"
+        echo "topic # IN hermes/"
     } >> /etc/mosquitto.conf
 fi
 
-echo "[Info] Start internal mqtt broaker"
+echo "[INFO] Start internal mqtt broker"
 mosquitto -c /etc/mosquitto.conf &
 
-# init snips config
-mkdir -p "$SNIPS_CONFIG"
-
+echo "[INFO] Checking for updated $ASSISTANT in /share"
 # check if a new assistant file exists
 if [ -f "/share/$ASSISTANT" ]; then
-    echo "[Info] Install/Update snips assistant"
-    unzip -o -u "/share/$ASSISTANT" -d "$SNIPS_CONFIG"
+    echo "[INFO] Install/Update snips assistant"
+    unzip -o -u "/share/$ASSISTANT" -d /usr/share/snips
 fi
-ln -s "$SNIPS_CONFIG/assistant/" /usr/share/snips/assistant
+
+sleep 2
```

Comment:
Why do we need to sleep 2?

Reply:
Sleep 2 is needed because on my rasp pi 3, the mosquitto broker wasn't starting fast enough for snips-watch to be able to connect to it. There is no retry method in that app but it is really useful to output the intent recognition to the logs. The logging does not show anything particularly useful otherwise.

Reply:
To clarify, without snips-watch running the only thing in the logs is when the asr stops and starts listening, so you can infer a hotword was detected but can't see anything else like what text was parsed or what intent was detected. I vaguely thought about some kind of 'wait until mosquitto starts' method but that would be fairly painful in bash and seemed like overkill

Reply:
If it's only needed for snips-watch, let's add the delay only to the background command: https://unix.stackexchange.com/a/102961